docker-compose.yml

```diff
@@ -9,10 +9,14 @@ services:
       - protectedplanet_node_modules:/ProtectedPlanet/node_modules
       - protectedplanet_import_data:/import_data
       - protectedplanet_bundler:/usr/local/bundle
-      - ${SSH_AUTH_SOCK}:${SSH_AUTH_SOCK}
-    network_mode: host
+      # - /run/host-services/ssh-auth.sock:/ssh-agent # Use this setting for Mac OS X and comment out ${SSH_AUTH_SOCK}:/ssh-agent below
+      - ${SSH_AUTH_SOCK}:/ssh-agent
+    ports:
+      - "3000:3000"
```

Comment:
Please change to "3000" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.